lib/ansible/modules/files/tempfile.py

```diff
@@ -55,6 +55,13 @@
   tempfile:
     state: file
     suffix: temp
+  register: tempfile_1
+
+- name: remove the temporary file
+  file:
+    path: "{{ tempfile_1.path }}"
+    status: absent
```

Comment:
As mentioned before, the correct parameter is `state ` not `status`. Please correct this example.